lib/util/exec/index.ts

```diff
@@ -141,6 +142,16 @@ async function prepareRawExec(
       ...(await generateInstallCommands(opts.toolConstraints)),
       ...rawCommands,
     ];
+  } else if (isHermit()) {
+    const hermitEnvVars = await getHermitEnvs(rawOptions);
```

Comment:
why do we need this?

Reply:
Hermit manages the packages and have the information about where they are located. This piece of information will be reflected in `hermitEnvVars`, which is a result of `hermit env`. By injecting these environment variables into the shell runs, project will use the binaries specified via Hermit.